src/emit/soroban/target.rs

```diff
@@ -193,9 +235,7 @@ impl<'a> TargetRuntime<'a> for SorobanTarget {
     }
 
     /// Prints a string
-    fn print(&self, bin: &Binary, string: PointerValue, length: IntValue) {
-        unimplemented!()
-    }
+    fn print(&self, bin: &Binary, string: PointerValue, length: IntValue) {}
```

Comment:
Should this function be doing nothing?

Reply:
When there is a runtime error, an overflow maybe, a call to this function is made. The ideal scenario is to make a call to Soroban's host function responsible for logging, but since this is out of scope for this PR, I left it out. 

A `//TODO` might be better here